src/Helpers.elm

```diff
@@ -62,12 +62,14 @@ classify formatted =
                 |> String.concat
                 |> String.all (\char -> char == '0')
     in
-        if onlyZeros then
-            Zero
-        else if formatted.original < 0 then
-            Negative
-        else
-            Positive
+    if onlyZeros then
+        Zero
+
+    else if formatted.original < 0 then
+        Negative
+
+    else
+        Positive
```

Comment:
This whole block (65-72) should be a level deeper in indentation, right? Right now the `if` and the `in` are aligned…  Or `elm-format` has new rules that I might have missed?

Reply:
Hum… just saw other blocks like that. Maybe `elm-format` has changed indeed… my bad.

Reply:
Correct